contents/handbook/people/feedback.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Getting Feedback
+sidebar: Handbook
+showTitle: true
+---
+
+## Feedback at PostHog
+
+Sharing and receiving feedback openly is _really_ important to us at PostHog. Part of creating a highly autonomous culture where people feel empowered is maintaining the most transparent and open flow of information that we can. 
+
+This includes giving feedback [to each other](/handbook/company/values#tread-on-toes), so we know we are working on the right things, in the right way. While giving feedback to a team member can feel awkward, especially if it is not positive or if you are talking to someone with more experience than you, we believe that it is an important part of [not letting others fail](/handbook/company/culture#dont-let-others-fail). 
+
+'Open and honest' != 'being an asshole' - we expect feedback to be direct, but shared with good intentions and in the spirit of genuinely helping that person and PostHog as a whole to improve. 
+
+## Performance Reviews
+
+In addition to informal day-to-day feedback, it is important that we enable team members to take a step back every so often and look at their performance in a wider context. This helps us to support a team member's growth and ensure it is aligned with PostHog's needs. 
+
+We have developed a simple performance review process to support this. We strongly believe that [it is _not_ a manager's responsibility](/handbook/company/management) to give their team members feedback, except in their capacity as any other individual contributor. 
+
+This process is intended to be self-serve. The People team will ensure the process is kicked off and recorded properly, but it is the individual team member's responsibility to run the process. If you need support, ask your manager, Eltje or Charles for help. 
+
+We currently run performance reviews every 6 months, based on your start date. We will probably need to change this cadence as we scale, but this feels appropriate for our current stage of growth. 
+
+### The Performance Review Process
+
+1. The People team will get an alert in Charlie HR every 6 months that someone is due to have their review.
+2. They will schedule a 1 hour feedback meeting for the team member, with an open invitation to _anyone_ at PostHog who wants to give feedback. It will be the team member's responsibility to encourage people to join if they would like their feedback. If you would be more comfortable with your manager facilitating the session, please ask them do so! A member of the people team may sit in on the occasional feedback meeting to see how well they are working. 
+3. The team member's manager then summarises the outcome of that meeting in this [document](https://docs.google.com/document/d/1fxP0w_gNno7Y-2Uxw4uSYCaJTpvZpDXiFZ7lFPXsDpw/edit?usp=sharing).
+4. The team member also writes up their own self-assessment in this [document](https://docs.google.com/document/d/1UbS9YkGDZsAhPsZmxRRI2g83ZuQzPwoQNQeJ7IGBm9I/edit?usp=sharing).
```

Comment:
minor comment made in the linked doc to resolve

Reply:
Left comment in there - I don't feel particularly strongly, so am happy to take Leadership out